spec/javascripts/controllers/account_controller_spec.coffee

```diff
@@ -214,3 +221,16 @@ do ->
       it 'calls ShortFormApplicationService.importUserData', ->
         scope._createAccountSubmitApplication()
         expect(fakeShortFormApplicationService.submitApplication).toHaveBeenCalled()
+
+    describe '$scope.isSale', ->
+      it 'calls ListingIdentityService.isSale', ->
+        scope.isSale(fakeListing)
+        expect(fakeListingIdentityService.isSale).toHaveBeenCalled()
+
+    describe '$scope.hasSaleAndRentalApplications', ->
```

Comment:
since the logic around checking to see if there are both rental and sale applications is slightly complicated, i'd like to see a test that's actually checking the different cases with fake applications or something.